kotlin/internal/toolchains.bzl

```diff
@@ -236,6 +237,14 @@ _kt_toolchain = rule(
                 "KOTLINBUILDER_REDUCED",
             ],
         ),
+        "experimental_track_class_usage": attr.string(
+            doc = "Whether to track used classes",
+            default = "off",
+            values = [
```

Comment:
why not use booleans here?

Reply:
yeah.. I should change that. a boolean should be fine, not planning to use a third value here anymore..